Formula/tesseract.rb

```diff
@@ -21,12 +21,11 @@ class Tesseract < Formula
   end
 
   depends_on "autoconf" => :build
-  depends_on "autoconf-archive" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "leptonica"
-  depends_on "libtiff"
+  depends_on "libarchive"
```

Comment:
Does this work with macOS `libarchive`?

Reply:
That `libarchive` will not be found by the build process because the expected `libarchive.pc` for `pkg-config` is missing.

Reply:
It might work as soon as Homebrew provides a `/opt/homebrew/Library/Homebrew/os/mac/pkgconfig/12/libarchive.pc` (in my build environment), like it is done for `libcurl.pc`.

Reply:
Ok, that's fine then. Thanks for clarifying.